contributors/devel/contributor-experience/github-magic.md

```diff
@@ -0,0 +1,101 @@
+
+  - Always use references about others PRs or issues. It is useful to identify that something has been addressed somewhere else.
+	
+### Opening a pull request
+Pull requests are useful in the following scenario:
+- You have done trivial fixes; that is broken links, typos, grammar mistakes and anything that is a clearly identifiable error.
+- You have done fixes to a contribution that has already been discussed on an issue.
+
+#### Pull requests advantages
+- Maintainers and contributors give feedback when commenting pull requests.
+- It can be labeled as “Work In Progress” if you’ll add more commits later.
+- They don’t represent a finished work.
+- If something is wrong about a PR, and a conflict that can harm the project integrity is identified, you can close it without affecting the project - itself.
+
+### Example of first comment when opening an issue or a PR:
+```
+Ref. #3064 #3097
+All files owned by SIG testing were moved from /devel to the new folder /devel/sig-testing.
+/sig contributor-experience
+/cc @parispittman @nikhita
+/kind cleanup
+/area developer-guide
+/assign @cblecker @spiffxp @timothysc
+```
+		
+#### What’s in that comment?
+  - Reference to another issues or PRs (#3064 #3097).
+  - A brief description of what’s done.
+  - SIG assignation with the command /sig contributor-experience. Learn more about [commands] and about [SIGs].
+  - Reviewers that may have interest on this specific issue or PR are specified with the /cc command.
+  - The /kind cleanup command add a label that categorizes issue or PR as related to cleaning up code, process, or technical debt. Learn more about [labels].
+  - The /area developer-guide command categorizes issue or PR as related to the developer guide.
+  - The command /assign assigns an assignee to the PR. Usually the file OWNER who will add the /approve label to the pull request.
+
+### About trivial fixes
+Trivial fixes or edits are always the best candidates for new contributors, and we believe this is a smooth way to introduce them to any subproject in the Kubernetes organization. However, it is more beneficial if we take the entire document which we are working with and give it a full review. Learn more about [trivial] fixes.
+
+### Working locally
+#### Branches
+Branches are you best friends when working locally. It is suggested to  create new branches to add new code. To create a new branch run:
+
+`git checkout -b myfeature`
+
+Learn more about the [Github workflow and branching].
+
+#### Squashing Commits
+The main purpose of squashing commits is to have a good readability of the git history. Grouping certain changes together before sharing allow us to have a very clean and easy to read documentation of the changes done. Usually this is done in last phase of a PR revision. Learn more about [squashing commits].
+
+
+[code of conduct]: https://github.com/kubernetes/community/tree/master/committee-code-of-conduct
```

Comment:
```suggestion
[code of conduct]: /code-of-conduct.md
```
This should point to the CoC instead of the CoC committee.

A few nits on the links below :x
The other links that reference the community repo should probably use reference links from the root and the git.k8s.io shortener.